iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/optional.inl

```diff
@@ -128,13 +137,18 @@ inline optional<T>& optional<T>::operator=(optional&& rhs) noexcept
         {
             construct_value(std::move(rhs.value()));
         }
+        else
+        {
+            // do nothing
+        }
         if (rhs.m_hasValue)
         {
             rhs.destruct_value();
         }
     }
     return *this;
 }
+// AXIVION ENABLE STYLE AutosarC++19_03-A13.3.1
```

Comment:
Remove. We should not disable rules for entire regions.

Reply:
Sure.